implementations/elixir/ockam/ockam/lib/ockam/transport/portal/outlet.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Ockam.Transport.Portal.OutLet do
+    use Ockam.Worker
+    alias Ockam.Message
+    alias Ockam.Transport.Portal.TunnelProtocol
+
+    require Logger
+
+    @impl true
+    def setup(options, state) do
+       msg = options[:init_message]
+       target_host = options[:target_host]
+       target_port = options[:target_port]
+       Logger.info("Starting outlet worker to #{target_host}:#{target_port}.  peer: #{inspect(msg.return_route)}")
```

Comment:
I we should access message fields consistently at least within a single module. We either use `Message.return_route(msg)` here or `msg.payload` next line.
Just so we don't confuse people who read the code later.

Reply:
Right, will fix this.  I was confused myself of what's more idiomatic on Elixir,   should I go with the `msg.payload`  way?